File: python/raindrops/raindrops.py
# nothing very clever here

def raindrops(number):
    rv=""
    if(number % 3 == 0):
        rv+="Pling"
    if(number % 5  == 0):
        rv+="Plang"
    if(number % 7  == 0):
        rv+="Plong"
    if(rv==""):
        rv=number
    return "%s"%rv

File: julia/raindrops/raindrops.jl
function raindrops(number)
    rv = ""
    rv = rv * (number % 3 == 0 ?  "Pling" : "")
    rv = rv * (number % 5 == 0 ?  "Plang" : "")
    rv = rv * (number % 7 == 0 ?  "Plong" : "")
    if rv == "" rv = string(number) end
    rv
end
